config-linux.md

```diff
@@ -147,7 +147,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/random`][random.4]
 * [`/dev/urandom`][random.4]
 * [`/dev/tty`][tty.4]
-* [`/dev/console`][console.4]
+* [`/dev/console`][console.4] is setup if terminal is enabled in the config
```

Comment:
Is “this device is missing” the usual way to handle “you don't have a current console”?  It seems like processes that expect `/dev/console` would expect something besides ENOENT for that (although maybe they handle both ENOENT and whatever “you don't have a current console” returns, and maybe “you don't have a current console” is ENOENT).